packages/react-vapor/src/components/toast/Toast.tsx

```diff
@@ -17,12 +17,18 @@ export interface IToastProps {
     isSmall?: boolean;
     className?: string;
     showInfoToken?: boolean;
+    onClose?: () => void;
     /**
      * @deprecated use children instead
      */
     content?: React.ReactNode;
+    /**
+     * @deprecated - can't find any instances in RV or Admin that use this but leaving in just case
+     */
     onRender?: () => void;
-    onClose?: () => void;
+    /**
+     * @deprecated - can't find any instances in RV or Admin that use this but leaving in just case
```

Comment:
This comment isn't useful for users of react-vapor
```suggestion
     * @deprecated - Use onClose instead
```